src/collection-view.js

```diff
@@ -14,6 +14,11 @@ Marionette.CollectionView = Marionette.View.extend({
   // flag for maintaining the sorted order of the collection
   sort: true,
 
+  // TODO: Discuss how to handle option to disable attach/beforeAttach on child view trees.
+  // These properties are a placeholder for now.
```

Comment:
Well, we can reproduce half of [this line in Region#show](https://github.com/ianmstew/backbone.marionette/blob/9907c2982fa57138d1a29596cd8dc41925ff32b2/src/region.js#L113) with a class option, but there isn't a direct equivalent to `showOptions` in any CollectionView method.  We might have to either not implement it, or add it to `render()`.  This goes back to my comment earlier that refactoring a CollectionView to be more like a Region might be beneficial, otherwise we will have to compromise on slight inconsistencies like this.